src/Components/LandingPage/sections/MostPickedSection.tsx

```diff
@@ -0,0 +1,195 @@
+      toast.error(error?.response?.data?.message);
+    })
+  }
+
+  return (
+    <div>
+
+      {/* sign in */}
+      <Modal
+        open={showState == 'unloginUser'}
+        onClose={handleClose}
+        aria-labelledby="modal-modal-title"
+        aria-describedby="modal-modal-description"
+      >
+        <Box sx={style}>
+
+          <FormControl component="form" onSubmit={handleSubmit(signIn)} sx={{ padding: "2rem", border: "none" }}>
+            <Grid containersx={{ bgcolor: "#ffffff", display: 'flex', alignItems: 'center', justifyContent: 'center' }}>
+              <Grid item xs={12}>
+                <Typography sx={{ color: '#000000', mr: 2, display: "flex", justifyContent: "flex-start", fontFamily: "Poppins", fontWeight: 300, marginBottom: '5px' }} >login first to enable to add rooms in your favorite...</Typography>
+                <Typography sx={{ color: '#000000', mr: 2, display: "flex", justifyContent: "flex-start", fontFamily: "Poppins", fontWeight: 300 }} >Email Address</Typography>
+                <TextField {...register('email', { required: true, pattern: /^[a-zA-Z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,4}$/ })} label="Email" type="email" margin="normal" fullWidth sx={{ bgcolor: "#F5F6F8", fontFamily: "Poppins" }} />
+                {errors.email && errors.email.type === 'required' && (<Typography sx={{ color: 'red' }}>email is required</Typography>)}
+                <Typography sx={{ color: '#000000', mr: 2, display: "flex", justifyContent: "flex-start", fontFamily: "Poppins", fontWeight: 300 }} >Password</Typography>
+                <TextField {...register('password', { required: true })} label="Password" type="password" margin="normal" fullWidth sx={{ bgcolor: "#F5F6F8", fontFamily: "Poppins" }} />
+                {errors.password && errors.password.type === 'required' && (<Typography sx={{ color: "red" }}> password is required</Typography>)}
+                <Typography sx={{ color: '#000000', mr: 2, display: "flex", justifyContent: "flex-start", fontFamily: "Poppins", fontWeight: 300 }} >
+                  <Link to={'/signup'} style={{ textDecoration: "none" }}>Creat an account!</Link>
+                </Typography>
+                <Button type="submit" variant="contained" fullWidth sx={{ bgcolor: '#3252DF', '&:hover': { bgcolor: '#2843CC' }, marginTop: "10px", fontFamily: "Poppins" }}>
+                  Sign In
+                </Button>
+              </Grid>
+
+            </Grid>
+          </FormControl>
+
+
+        </Box>
+      </Modal >
+
+      <Grid container columnSpacing={{ xs: 1, sm: 2, md: 3 }} sx={{ justifyContent: "center", padding: "3rem 0" }}>
+
+        <Grid item xs={10} >
+          <Typography id="modal-modal-title" variant="h6" component="h2" sx={{ marginBottom: "1rem" }}>
+            Houses with beauty backyard
+          </Typography>
+        </Grid>
+
+        {/* main section */}
+        <Grid item xs={10} container sx={{ justifyContent: "space-between" }}>
```

Comment:
You need to adjust the structure and make it responsive. You can have a look at the MUI grid and how to customize it in different screen sizes.
https://mui.com/material-ui/react-grid/